VetClinicPublic/src/VetClinicPublic/App_Start/BundleConfig.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Web.Optimization;
+
+namespace VetClinicPublic.Web
+{
+    public class BundleConfig
```

Comment:
Is this still in modern "File New Web Project" templates for .NET 5? If not, let's remove it. I'm not sure we need jquery, etc. in the public web site, either.

Reply:
Oh wait, we probably need this to use the Kendo stuff as a workaround, right?